vue-frontend/src/pages/jobs/[id].vue

```diff
@@ -291,21 +291,32 @@ export default {
       });
     },
     setBulletsAndIconsInDescription() {
-      let descriptionTitles = document.querySelectorAll("#description > h2");
-      descriptionTitles.forEach((descriptionTitle) => {
-        let bullet = document.createElement("span");
-        bullet.className = "bullet";
-        descriptionTitle.prepend(bullet);
-      });
-      let descriptionLists = document.querySelectorAll(
-        "#description > ul > li > div"
-      );
-      descriptionLists.forEach((descriptionList) => {
-        let fa = document.createElement("span");
-        fa.className = "fas fa-chevron-right";
-        fa.innerHTML = "&nbsp;";
-        descriptionList.prepend(fa);
-      });
+      let descriptionTitles = this.job.description.match(/<h2>(.*?)<\/h2>/g);
+
+      for (let i = 0; i < descriptionTitles.length; i++) {
+        if (descriptionTitles[i]) {
+          let title = descriptionTitles[i].match(/<strong>(.*?)<\/strong>/g)[0];
+          this.job.description = this.job.description.replace(
+            descriptionTitles[i],
+            '<h2><span class="bullet"></span>' + title + "</h2>"
```

Comment:
Do we need to append the` h2` tag here? as we are replacing the child element that is `strong`

Reply:
yes, we need to append h2 tag. because  replace that h2 tag `descriptionTitles[i]`